Sources/AST/ASTDumper.swift

```diff
@@ -454,6 +455,14 @@ public class ASTDumper {
     }
   }
 
+  func dump(_ typeConversionExpression: TypeConversionExpression) {
+    writeNode("TypeConversionExpression") {
+      self.dump(typeConversionExpression.expression)
+      self.writeLine(typeConversionExpression.kind.description)
```

Comment:
I still think this should read `token: as` in the AST dump, for consistency with the other nodes.

Reply:
But as can come in three flavors. Those are not strictly tokens themselves 
as is a token 
We can have as? Which is two tokens. as! Is the same. So I didn’t see token: as being correct for this

Reply:
`kind: as` ?